index.js

```diff
@@ -160,12 +164,26 @@ Liftoff.prototype.buildEnvironment = function (opts) {
   var configPathOverride = arrayFind(Object.keys(config), function (key) {
     var cfg = config[key];
     if (Object.prototype.hasOwnProperty.call(cfg, configName)) {
-      if (typeof cfg[configName] === "string") {
+      if (isString(cfg[configName])) {
         return cfg[configName];
       }
     }
   });
 
+  var additionPreloads = arrayFind(Object.keys(config), function (key) {
+    var cfg = config[key];
+    if (Object.prototype.hasOwnProperty.call(cfg, 'preload')) {
+      if (Array.isArray(cfg.preload)) {
+        if (cfg.preload.every(isString)) {
+          return cfg.preload;
+        }
+      }
+      if (isString(cfg.preload)) {
```

Comment:
It's minor, but wouldn't it be better to use `else if` for the `isString` check?

Reply:
I don't think it's necessary. The only time this would be an extra check is if the value is an array without all strings, which makes it entirely invalid anyway